ALevel HW5/Action.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ALevel_HW5
+{
+    public class Actions
+    {
+        private Logger logger = Logger.Instance;
+
+        public Result Method1()
```

Comment:
You should use more descriptive naming. Like Success, Skipped, Failed smth with these words